src/api/app/models/bs_request.rb

```diff
@@ -97,7 +130,7 @@ def check_supersede_state
   end
 
   def superseding
-    BsRequest.where(superseded_by: id)
+    BsRequest.where(superseded_by: self.number)
```

Comment:
no, we could convert it, but it would be some more work since it goes through the event system.